packages/devextreme-cli/src/templates/vue-v3/sample-pages/home-page.vue

```diff
@@ -177,17 +179,37 @@
 .devextreme-logo {
   width: 255px;
   height: 60px;
+
+  > path {
+    fill: var(--devextreme-logo-color);
```

Comment:
I dont see any difference between using css vars and scss vars in this case. Maybe use scss here to maintain same style of vars?